frontend-public/src/styles/components/Loading.scss

```diff
@@ -14,3 +14,17 @@
     @include center(true, true);
   }
 }
+
+.loading-screen--small {
+  min-height: 500px;
+  width: 100%;
+  #spinner {
+    // offset size of spinner to center
+    left: 45% !important;
```

Comment:
Can you explain why it needs to be offset by 5%?

Reply:
The size of the spinner is based on `font-size` not on `width/height`, and because of that when you increase the `font-size` the icon does not re-center. 
I was forcing it to re-center here based on the larger `font-size`